packages/client/src/client/components/Form/TextInput.tsx

```diff
@@ -0,0 +1,59 @@
+import React from "react"
+import styled from "styled-components"
+
+const TextInputText = styled.input(
+  ({ theme }) => theme.newTheme.textStyles["Text md/Regular"],
+)
+
+const _TextInput = styled(TextInputText)<Partial<HTMLInputElement>>`
```

Comment:
ideally not underscore-prefixed, unless there is some special reason for that in this context